tendrl/commons/objects/node/atoms/check_service_status/__init__.py

```diff
@@ -0,0 +1,142 @@
+        tags = list(node_context.tags)
+        service_status = True
+        if "tendrl/monitor" in tags or \
+                "tendrl/integration/monitoring" in tags:
+            # check neccessary service status in server
+            for service_name in TENDRL_SERVICES["server"]:
+                service = get_service_status(
+                    service_name
+                )
+                if not service.running:
+                    if len(service.error) > 0:
+                        msg = ("Failed to check status of %s "
+                               "on server-node. Error: %s" % (
+                                   service_name,
+                                   service.error
+                               ))
+                    else:
+                        msg = ("Service %s is not running on a server-node, "
+                               "Please start it manually or check the log "
+                               "file to figure out the exact "
+                               "problem" % service_name)
+                    logger.log(
+                        "error",
+                        NS.get("publisher_id", None),
+                        {
+                            "message": msg
+                        },
+                        job_id=self.parameters['job_id'],
+                        flow_id=self.parameters['flow_id']
+                    )
+                    service_status = False
+        else:
+            # check neccessary service status in storage nodes
+            for service_name in TENDRL_SERVICES["storage_node"]:
+                service = get_service_status(service_name)
+                if not service.running:
+                    if len(service.error) > 0:
+                        msg = ("Failed to check status of %s "
+                               "on %s. Error: %s" % (
+                                   service_name,
+                                   NS.node_context.fqdn,
+                                   service.error
+                               ))
+                    else:
+                        msg = ("Service %s is not running on %s, Please "
+                               "start it manually or check the log file to "
+                               "figure out the exact problem" % (
+                                   service_name,
+                                   NS.node_context.fqdn
+                               ))
+                    logger.log(
+                        "error",
+                        NS.get("publisher_id", None),
+                        {
+                            "message": msg
+                        },
+                        job_id=self.parameters['job_id'],
+                        flow_id=self.parameters['flow_id']
+                    )
+                    service_status = False
+            node_list = self.parameters['Node[]']
+            if service_status and len(node_list) > 1:
```

Comment:
Add a comment sating that the monitoring integration service check should happen from the bootstrap node only